torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12774,6 +12813,13 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            supports_out=False),
+    OpInfo('slice',
```

Comment:
Nice!

Reply:
Test failures are real, though, because `torch.slice` doesn't exist. 

You can run many of the tests locally with `python test/test_ops.py -v -k slice` to see what they're doing.

I think this OpInfo needs to define its `op` metadata. This is sometimes done to wrap an operation and set a seed so it can be used for reproducible testing, like with `torch.uniform`: 

https://github.com/pytorch/pytorch/blob/90fa744c09fdbf8a2e7747ea82823714b0ee7e04/torch/testing/_internal/common_methods_invocations.py#L8164

It's also done when there's a torch method but not a torch function for an operation, like with `contiguous`: 

https://github.com/pytorch/pytorch/blob/90fa744c09fdbf8a2e7747ea82823714b0ee7e04/torch/testing/_internal/common_methods_invocations.py#L8814

And for more exotic cases, like creating a jiterated op:

https://github.com/pytorch/pytorch/blob/90fa744c09fdbf8a2e7747ea82823714b0ee7e04/torch/testing/_internal/common_methods_invocations.py#L15051